stdlib/public/runtime/Metadata.cpp

```diff
@@ -2589,15 +2612,40 @@ swift::swift_initClassMetadata(ClassMetadata *self,
 #if SWIFT_OBJC_INTEROP
 void
 swift::swift_updateClassMetadata(ClassMetadata *self,
-                                 ClassMetadata *super,
                                  ClassLayoutFlags layoutFlags,
                                  size_t numFields,
                                  const TypeLayout * const *fieldTypes,
                                  size_t *fieldOffsets) {
+#ifndef NDEBUG
```

Comment:
This is almost identical to the initClassMetadata() case, can you clean it up in a follow-on PR?

Reply:
Yes, absolutely.